superchain/extra/addresses/addresses.json

```diff
@@ -63,7 +72,10 @@
     "ProxyAdminOwner": "0xE75Cd021F520B160BF6b54D472Fa15e52aFe5aDD",
     "SystemConfigOwner": "0x23BA22Dd7923F3a3f2495bB32a6f3c9b9CD1EC6C",
     "Guardian": "0x7a50f00e8D05b95F98fE38d8BeE366a7324dCf7E",
-    "Challenger": "0x45eFFbD799Ab49122eeEAB75B78D9C56A187F9A7"
+    "Challenger": "0x45eFFbD799Ab49122eeEAB75B78D9C56A187F9A7",
+    "UnsafeBlockSigner": "0x93A14E6894eEB4FF6a373E1Ad4f498c3a207afe4",
+    "BatchSubmitter": "0x4e6BD53883107B063c502dDd49F9600Dc51b3DDc",
+    "Proposer": "0xe9e08A478e3a773c1B5D59014A0FDb901e6d1d69"
```

Comment:
Are these changes intentionally included in this PR? They keep creeping into PRs :laughing:

Reply:
Fixed here [3bdd1b9](https://github.com/ethereum-optimism/superchain-registry/pull/337/commits/3bdd1b96a50f40aa2db0e6c20fabfadcb81948a3)